api/v1/server/authz/middleware.go

```diff
@@ -87,6 +116,23 @@ func (a *AuthZ) authorize(c echo.Context, r *middleware.RouteInfo) error {
 	return nil
 }
 
+var restrictedWithBearerToken = []string{
```

Comment:
update the admin and owner permissions to allow for API token creation/deletion/listing, but don't allow members to do these commands, because that could elevate the privileges.